week6/assignment2_1/includes/ex.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef EX_HPP
+#define EX_HPP
+#include <iostream>
+#include <limits>
+
+class Positive_integers {
+    public:
+        int number{0};
+        Positive_integers() = default; 
+        Positive_integers(int num);
+        void print_number();
+        int add(Positive_integers obj);
+        int subtract(Positive_integers obj);
+        int max_number{std::numeric_limits<int>::max()};
```

Comment:
Since this is the same for all instances, it _could_ be made `static`.  Either way, it should be a constant.

My suggestion: `constexpr static int max_numer{std::numeric_limits<int>::max()};`